src/redux/modules/chronicles.js

```diff
@@ -13,3 +13,24 @@ export const actions = {
   userInactive,
 };
 
+/* Reducer */
+const initialState = {
+  actionsCount: 0,
+  lastAction: null,
+};
+
+const onAction = (state, action) => {
+  state.actionsCount = state.actionsCount + 1;
+  state.lastAction = action;
+  return state;
+};
+
+export const reducer = handleActions({
+  ['FETCH_RECOMMENDED_SUCCESS']: onAction,
```

Comment:
handleActions() is a wrapper to `immer`, a library to work with immutable state. It calls your function with two parameter: `draft `and `payload`. `draft` means a temporary storage, immer will track its changes, will create a new immutable state accordingly and will assign it to state. `payload` is `action` in your case, but may be just anything. Please use the same naming convention.